bolt-service/README.md

```diff
@@ -0,0 +1,81 @@
+# Bolt Service
+
+This project creates a docker image for the bolt service.   
+Currently the service generates test certificates that are copied into the
+image.   For production use these should be overwritten with your own certs,
+see details below.
+
+
+## Building
+Run make to build the project, this generates test certs and creates a docker
+image called puppet/bolt-service:latest
+
+```
+make
+```
+
+## Running the service
+You can run the service by creating a new container.  N.B by default the Service
+listens on port 8144, so you must ensure it is exposed.
```

Comment:
We changed it! It listens on `62658` (call 1-800-6-BOLT if you ever need to remember it :smile: )